Tests/Clappr_Tests/Classes/Extensions/UIImageExtension.swift

```diff
@@ -0,0 +1,9 @@
+import UIKit
+
+extension UIImage {
+    func isEqualTo(image: UIImage) -> Bool {
```

Comment:
Hey, this is not a change requirement, but UIImage conforms to the `Equatable` protocol, so we have already `isEqual` method. What do you think about overriding it and implementing it like this:

```swift 

extension UIImage {
    override open func isEqual(_ object: Any?) -> Bool {
        guard let image = object as? UIImage,
         let lhs: Data = UIImagePNGRepresentation(self),
         let rhs: Data = UIImagePNGRepresentation(image) else {
            return super.isEqual(object)
        }

        return lhs == rhs
    }
}
```

Reply:
Done.